LayoutTests/platform/ios/TestExpectations

```diff
@@ -27,6 +27,22 @@ fast/events/ios [ Pass ]
 fast/events/ios/ipad [ Skip ]
 fast/forms/ios [ Pass ]
 fast/forms/ios/ipad [ Skip ]
+fast/forms/switch [ Pass ]
+
+# More work required for these tests to run on iOS
```

Comment:
Are all of these due to a lack of painting code, or is there another issue?

Reply:
`fast/forms/switch/click-animation-twice.html` only passes if I increase the final timeout from 50 to 75.